Src/Orbiter/Orbiter.cpp

```diff
@@ -16,7 +16,7 @@
 #include <stdio.h>
 #include <time.h>
 #include <fstream>
-#include <strstream>
+//#include <sstream>
```

Comment:
Should it be commented out?

Reply:
No, actually it should have been removed 😉 Since nothing in `Orbiter.cpp` is using any of that.
Unfortunately it was left as "commented out". I'll fix (remove) that!